src/HtmlGenerator/Pass1-Generation/ProjectGenerator.cs

```diff
@@ -221,15 +221,7 @@ private bool IsCSharp
             }
         }
 
-        private bool IncludeDocument(Document document)

```

Comment:
The reason I wrote this method like this is it's future proof - it's easy to add additional checks to such a method.

Granted, this method may never change in the future, in which case your refactoring is OK. So I'm on the fence about this one.